docs/data/charts/bars/bars.md

```diff
@@ -8,16 +8,28 @@ components: BarChart, BarChartPro, BarElement, BarPlot, ChartsGrid, BarLabel
 
 <p class="description">Bar charts express quantities through a bar's length, using a common baseline.</p>
 
+{{"component": "@mui/docs/ComponentLinkHeader", "design": false}}
+
+## Introduction
+
+TK
+
 ## Basics
 
-Bar charts series should contain a `data` property containing an array of values.
+```tsx
+import { BarChart } from '@mui/x-charts/BarChart';
+```
+
+### Data
+
```

Comment:
I'm not fan of this aproach

- I don't know what to put in "Introduction" that is not already in the description.
- I like having "Basics" (maybe "Basic bar chart" would be better) as a first section with just the strict minimal explanation about props (only the required ones) and a straightforward demo. Such that as a use, I can expand and copy the code.